lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -84,6 +84,13 @@ feature:
   config_get_token: '/^feature bgp$/'
   config_set: "<state> feature bgp"
 
+# This is a multi-provider, enable-only feature. Do not disable from provider.
+feature_nv_overlay_evpn:
```

Comment:
We already have this defined in bgp_af.yaml so no need to duplicate it here. This is related to my comment for line 520 in bgp.rb.